custom_filter.py

```diff
@@ -1,4 +1,4 @@
-def custom_filter(function, iterable):
+def custom_filter(function: callable, iterable: iter) -> list:
```

Comment:
iter is not a type, you can use this "cheat sheet" when uncertain. 
https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html